knowledge-base/basic-authentication-with-fiddler-everywhere.md

```diff
@@ -18,55 +18,56 @@ res_type: kb
+The following steps provide an overview of the procedure used to create a basic authentication request:
 
-![Composer URL Field](../images/kb/url-field-of-composer-with-http-method.png)
+1. Select the [Composer]({%slug composer-tab%}) tab.
+1. Set the HTTP/HTTPS method to __GET__ and add the URL in the URL field.
+1. Create an object in the request __Body__ and the `user` and `passwd` variables and their values. In this object, `user` and `passwd` are the predefined variables for the __Basic Authentication__.
+1. Add an `Authorization` key to the header:
+    * Encode the value of the `<username>:<password>` string with a [__Base64 converter__](https://www.base64decode.org/) and note the encoded value.
+    * In the __Headers__ tab, set the __Key__ to `Authorization` and add the encoded `<username>:<password>` under __Value__.
+    * Add a __Description__ for the key. Click the __tick__ to add the `Authorization` key.
 
-In the above image, the specified URL is [https://httpbin.org/basic-auth/user1/pass1](https://httpbin.org/basic-auth/user1/pass1). Here __basic-auth__ determines that the __Authentication__ taking place is Basic. The next two fields __user1__ and __pass1__ are the values of `user` and `passwd` variables, respectively.
+## Basic Authentication Request Example
 
-## Step 2
+1. In the [Composer]({%slug composer-tab%}) tab, set the HTTP/HTTPS method to __GET__ and add [https://httpbin.org/basic-auth/user1/pass1](https://httpbin.org/basic-auth/user1/pass1) in the URL field.
 
-Create an object inside the __Body__ tab. The predefined variables for the __Basic Authentication__ are `user` and `passwd`. Set the values for both variables.
+   ![Composer URL Field](../images/kb/url-field-of-composer-with-http-method.png)
 
-![JSON Body](../images/kb/body-of-composer-with-username-password.png)
+   The [https://httpbin.org/basic-auth/user1/pass1](https://httpbin.org/basic-auth/user1/pass1) URL includes the following fields:
 
-In the above image, the values of the `user` and `passwd` variables are __user1__ and __pass1__, respectively.
+      * The [httpbin.org](https://httpbin.org/) HTTP request and response service. 
+      * The authentication scheme [__basic-auth__](https://tools.ietf.org/html/rfc7617).
+      * The values of the `user` and `passwd` variables: __user1__ and __pass1__.
 
-## Step 3
+1. Create a JSON object in the __Body__ tab:
 
-Once you create the object in the __Body__, go to the __Headers__ tab, where you can see a predefined key `User-Agent` added from Fiddler Everywhere. Now, let's add the Authorization key to the header. For that:
+   ![JSON Body](../images/kb/body-of-composer-with-username-password.png)
 
-- Encode the values of the variables set in the JSON format. Encode the texts with simple __Base64 converters__. The format of the text should be `<username>:<password>`. The encoded value of __user1:pass1__ is `dXNlcjFwYXNzMQ==`.
```

Comment:
The encoded value isn't correct and is different from the value on the screenshot. Maybe this is one of the reasons for this article's lower score.